### PR TITLE
Fix the staging directory path in copying from GCS

### PR DESCRIPTION
**Pull request by sb2nov**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`.
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

R: @aaltay PTAL

### CODE REVIEW

**Comment by aaltay** [comment]
Could you try to improve functions a bit, perhaps update comments. These two functions are using not clearly defined arguments.

**Comment by sb2nov** [comment]
Can we decouple the fix and the refactoring as I want to read the code in a bit more detail.

File: sdks/python/apache_beam/runners/dataflow/internal/dependency_test.py [hunk 1]
```diff
@@ -31,6 +31,16 @@
 from apache_beam.options.pipeline_options import SetupOptions
 
 
+# Protect against environments where bigquery library is not available.
```
Comment (aaltay) [inline sdks/python/apache_beam/runners/dataflow/internal/dependency_test.py]: How is this related to bigquery?

Comment (sb2nov) [inline sdks/python/apache_beam/runners/dataflow/internal/dependency_test.py]: Fixed

**Comment by aaltay** [comment]
LGTM. Thank you.

Please file a JIRA for refactoring the code, if you do not have time now.
